scripts/upload_test_results/README.md

```diff
@@ -0,0 +1,40 @@
+# Auto-upload daily test results
+
+## Python script to automatically upload your HW Daily Test Results to the Github test_results repo 
+https://github.com/zephyrproject-rtos/test_results
+
+**To customize the script and run it on your machine, please modify code in Step 1 and Step 2 (Use Search to find that comments in the code of the script).**
+
+### Script description:
+1. The script creates a git directory, if a directory already created, script updates it (step 2-3) (-g argument)
+   * Fetches code from the source repo master branch zephyrproject-rtos/test_results
+   * Pushes updates to the origin repo master branch, in my case it is maksimmasalski/test_results
+2. Downloads versions.json file and finds the proper commit name for existing test results.
```

Comment:
This opens the door to potential desynchro between the actual results and the published version.
Of course one can say that "this should not happen because there is usually 24 hours between to published versions" but:
- there is no policy to detect the change of version, so a non reg session could be run some time after the publishing time (eg each day at 10PM, which can be any time vs "publishing time zone")
- delay between 2 published tags may evolve in time or depending on conditions (eg bi-daily during release periods).

It would be safer to provide the tested tag as argument of the script

Reply:
Do you mean "tested" that already was reviewed by user manually here ?https://testing.zephyrproject.org/daily_tests/versions.json

Reply:
Sorry, I 

> Do you mean "tested" that already was reviewed by user manually here ?https://testing.zephyrproject.org/daily_tests/versions.json

I actually meant the tag that was used to provide the result we want to upstream.
There is no guarantee this is the last one from https://testing.zephyrproject.org/daily_tests/versions.json at the time upload_test_results.py is invoked.

Reply:
Yes, it can be the not last one. So initially I had code that searches in file specified commit tag, now I updated code that takes the last one. As I understand, have to keep search for the specified commit tag, and not take the last one like I made now.